lib/kino/input.ex

```diff
@@ -133,6 +172,13 @@ defmodule Kino.Input do
     end
   end
 
+  defp assert_debounce_value!(value) do
+    unless is_nil(value) or value == "blur" or (is_number(value) and value >= 0) do
```

Comment:
We never expect `nil`

```suggestion
    unless value == :blur or (is_number(value) and value >= 0) do
```

Reply:
👍